Scripts/ue4ss_git.lua

```diff
@@ -0,0 +1,79 @@
+import("core.base.option")
+import("devel.git")
+import("devel.git.submodule")
+import("core.base.semver")
+
+local ue4ss_url = "https://github.com/UE4SS-RE/RE-UE4SS.git"
+local ue4ss_dir = "RE-UE4SS"
+
+function main()
+    local remote = option.get("remote")
+    local update = option.get("update")
+
+    if not (remote and update) then
+        print("No arguments. Run 'xmake ue4ss --help'")
+        os.exit()
+    end
+    
+    try_clone_ue4ss()
+
+    -- Special handling if the user wants to find the 'latest' release tag.
+    if remote == "latest" then
+        print("Attempting to find latest release tag...")
+        local tags = git.tags(ue4ss_url)
+        local latest_tag
+        for _,tag in ipairs(tags) do
+            if semver.is_valid(tag) then
+                if not latest_tag then
+                    latest_tag = tag
+                end
+
+                if semver.compare(tag, latest_tag) == 1 then
+                    latest_tag = tag
+                end
+            end
+        end
+
+        -- Special logic to ensure we don't check out a tag that doesn't have xmake.
+        -- This would put the user's local repo into an unrecoverable state.
+        if latest_tag then
+            if not semver.satisfies(latest_tag, ">3.0.1") then
```

Comment:
Just to double check, this excludes 3.0.1 right? Just double checking as you have 3.0.1 as an example in the readme which shouldn't be valid.